manim/mobject/probability.py

```diff
@@ -27,6 +27,24 @@
 
 
 class SampleSpace(Rectangle):
+    """A sample space.
+
+    Examples
+    --------
+    .. manim:: ExampleSampleSpace
+        :save_last_frame:
+
+        class ExampleSampleSpace(Scene):
+            def construct(self):
+                poly1 = SampleSpace(stroke_width=15,fill_opacity=1).move_to([-1,0,0])
+                poly2 = SampleSpace(width = 5, height = 3, stroke_width=5,fill_opacity=0.5).move_to([4,0,0])
+                poly3 = SampleSpace(width = 2, height = 2, stroke_width=5,fill_opacity=0.1).move_to([-4.5,0,0])
```

Comment:
Can you follow the instructions in [#hackthod-guidlines](https://discord.com/channels/581738731934056449/846805485529464832/849962667568529428)? ie, use the manimator bot(`!mdocstring` command) to create doc strings. I am asking that because it runs through an automatic formatted as explained in the guide lines.

Reply:
Hi, I used the mdocstring command and the suggested documentation looks like this:
![image](https://user-images.githubusercontent.com/18205576/120916560-f3711c80-c6c7-11eb-8408-e7e95ec15662.png)

It's the same as the current example. 
I am not sure what I need to change. Could you please help @naveen521kk?

![image](https://user-images.githubusercontent.com/18205576/120916598-1dc2da00-c6c8-11eb-9e83-021536ec9fa6.png)

Reply:
It's same but not exactly same. The code from the bot contains some formatting which is missing here. Possibly simply pasting that here should fix.